recipes/google-weather-tools/meta.yaml

```diff
@@ -0,0 +1,89 @@
+{% set name = "google-weather-tools" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: db9e8566091b72a1f20813f043b356590dc6aefd9f5cc5c983c0cbc9d4faf0e5
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python=3.8.13
```

Comment:
These dependencies are too strict, should not be in `build` and generally don't match the format we use in recipes. I suggest you try generating this recipe using `grayskull`.

Reply:
Hey @xhochy , thanks for looking into this. We have updated the meta.yaml to fix the format. Regarding the fixed library versions, this distribution is intended to replace PyPi as the official channel for production Dataflow pipelines and some of our 3rd party dependencies are very sensitive to dependency versions.

Reply:
We've fixed this in the new version which is based on `grayskull`